### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by bronzdoc**
Adds missing issue template

### CODE REVIEW

**Review by rdormer — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+**Before we start...:**
+
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+
+**Version, Branch, or Commit:**
+
+Inform what version, branch, commit of Skunk you are using.
```
Comment (etagwerker) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: @bronzdoc Should say Pecas

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**Before we start...:**
+
+- [ ] I checked the documentation and didn't find this feature
+- [ ] I checked to make sure that this feature has not already been requested
+
+
+**Branch/Commit:**
+
+Inform what branch/commit/version of "Skunk" you are using.
```
Comment (etagwerker) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]: @bronzdoc Should say Pecas

**Review by etagwerker — CHANGES_REQUESTED** [review]
@bronzdoc I noticed a couple copy/paste issues 👍

**Comment by bronzdoc** [comment]
Addressed the issues @etagwerker, thanks

**Pull request merged**